### ISSUE TITLE
STATUS_STACK_OVERFLOW when attempting to run examples

### ISSUE THREAD

**Issue by thomasmarschall** [Wazner/mssql-browser#1]
If I do:
```
git clone https://github.com/Wazner/mssql-browser.git
cd mssql-browser
cargo run --example="async-std" --features="async-std"
```

I get this output:
```
Listening...

thread 'main' has overflowed its stack
error: process didn't exit successfully: `target\debug\examples\async-std.exe` (exit code: 0xc00000fd, STATUS_STACK_OVERFLOW)
```

**Comment by Wazner** [comment]
Fixed by #2 

Thank you for your contribution!

**Comment by Wazner** [comment]
I have published a new version of the crate where this bug is fixed.
